Packages/com.llealloo.audiolink/Editor/Scripts/AudioLinkAssetManager.cs

```diff
@@ -37,7 +40,7 @@ private static void ReimportPackage()
         }
 
 #if !AUDIOLINK_STANDALONE
-        [MenuItem("Tools/AudioLink/Open AudioLink Example Scene")]
+        [MenuItem("AudioLink/Open AudioLink Example Scene")]
```

Comment:
looks like you have an unintended change here after your last pr went to master?

Reply:
Ah yea. Fixed.